src/v2/Apps/Order/Components/ShippingQuotes.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react"
+import { BorderedRadio, BoxProps, Flex, RadioGroup, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShippingQuotes_shippingQuotes } from "v2/__generated__/ShippingQuotes_shippingQuotes.graphql"
+import { compact } from "lodash"
+
+export const shippingQuoteDescriptions = {
+  ground: "Delivers in 3-5 days once shipped.",
+  "second day air": "Delivers in 2 business days once shipped.",
+  "next day air": "Delivers the following business day once shipped. ",
+  select:
+    "Consolidated transportation including blanket wrap and transit packing with white glove handling at collection and delivery.",
+  premium:
+    "Specialized climate controlled transportation and museum quality packing, handled by trained technicians from wall to wall.",
+}
+
+export interface ShippingQuotesProps extends BoxProps {
+  onSelect: (shippingQuoteId: string) => void
+  shippingQuotes: ShippingQuotes_shippingQuotes
+  selectedShippingQuoteId?: string
+}
+
+export const ShippingQuotes: React.FC<ShippingQuotesProps> = ({
+  onSelect,
+  shippingQuotes,
+  selectedShippingQuoteId,
+  ...rest
+}) => {
+  const quotes = compact(shippingQuotes?.map(quote => quote.node))
+
+  if (!quotes || !quotes.length) {
+    return null
+  }
+
+  return (
+    <RadioGroup
+      {...rest}
+      onSelect={onSelect}
+      defaultValue={selectedShippingQuoteId}
+    >
+      {quotes.map(shippingQuote => {
+        const { id, name, price, tier } = shippingQuote
+        const shippingQuoteName = name || tier
```

Comment:
non-blocking, info: Hmm, this feels like it might cause mishaps in the future but I'm not close to the work. Is this always going to be the case where there's no name for `select` and `premium` tiers? What if Arta suddenly starts returning a name for quotes in those tiers?

Reply:
In a case when Arta starts returning a name for quotes according to current logic we will show only the name and price(without description). 

I think it's okay to use the name because if for example, we receive 2 quotes with a `premium` tier (quotes with `Premium 1` and `Premium 2` names) that will be useful to identify each quote(by name in this case). Also since we use our own quote descriptions it's not a good idea to my mind to use the current `premium` description for `Premium 1` and `Premium 2` quotes because they may differ a lot.